docs/source/coroutines.rst

```diff
@@ -107,7 +119,14 @@ Other tasks can be used in an await statement to suspend the current task until
 Tasks can be killed before they complete,
 forcing their completion before they would naturally end.
 
-.. code-block:: python3
+.. code-block:: python
+
+    from math import isclose
+
+    import cocotb
+    from cocotb.clock import Clock
+    from cocotb.triggers import RisingEdge, Timer
+    from cocotb.utils import get_sim_time
```

Comment:
Yuck. Is there any way to prevent this? Maybe if this were a standalone file?

Reply:
Nvm, it looks like codeautolink handles this with "invisible imports". Definitely do that.